project/build.scala

```diff
@@ -1,11 +1,12 @@
 import sbt._
 
+
 object Build extends sbt.Build {
-  lazy val geospace= Project( "geospace-proj", file(".") ).
-    settings(BuildSettings.buildSettings : _*).
+  lazy val geospace = Project( "geospace-root", file("."), settings = BuildSettings.rootSettings).
+    settings(BuildSettings.buildSettings: _*).
```

Comment:
Is .settings(.....) additive, or is it just overwriting the first settings = ..... ?